nav2_util/src/string_utils.cpp

```diff
@@ -45,4 +51,117 @@ Tokens split(const string & tokenstring, char delimiter)
   return tokens;
 }
 
+
+/** @brief Parse a vector of vector of floats from a string.
+ * @param input
+ * @param error_return
+ * Syntax is [[1.0, 2.0], [3.3, 4.4, 5.5], ...] */
+std::vector<std::vector<float>> parseVVF(const std::string & input, std::string & error_return)
```

Comment:
Please verify for me: this is an exact  copy paste from costmap 2d, right?

Reply:
Yes!